packages/vue/LICENSE.md

```diff
@@ -0,0 +1,161 @@
+                                 Apache License
```

Comment:
There was no license file in that directory. This is part of the build that copies it. I restored the root-level license's whitespace so it will get copied to each package's directory.

Reply:
thanks